status/format_test.go

```diff
@@ -67,9 +70,12 @@ func newTaskRunWithAppStudioTestOutput(name string, startTime time.Time, complet
 var _ = Describe("Formatters", func() {
 
 	var taskRuns []*helpers.TaskRun
+	var pipelineRun *tektonv1.PipelineRun
+	var hasApp *applicationapiv1alpha1.Application
 
 	BeforeEach(func() {
 		now := time.Now()
+		os.Setenv("CONSOLE_URL", "https://definetly.not.prod/preview/application-pipeline/workspaces/%s/applications/%s/pipelineruns/%s")
```

Comment:
this won't work, workspace can be differnet than namespace

Reply:
Updated so it now uses ns in link.